### DISCUSSION TITLE
committer: Cody Peterson

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/governance#2] [Committer] [answered]
I would like to nominate Cody Peterson (@lostmygithubaccount) for Ibis committership.

Cody worked tirelessly on moving us over to Quarto, and is heavily invested in the success of Ibis. 

He has contributed to drastic improvements to our documentation, helped with issue triage and implemented Delta Lake IO support for a few backends.

**Comment by gforsyth** [comment]
I'm a +1 on this

**Reply by jcrist** [reply]
I'm also +1 on this

**Comment by cpcloud** [comment, marked as answer]
+1

With 3 votes, the motion carries! Welcome to the team @lostmygithubaccount!

**Reply by lostmygithubaccount** [reply]
I am honored :) thanks
